### PR TITLE
Revert "Create README.md"

### PR DESCRIPTION
**Pull request by Newlife08**
Reverts Newlife08/bc1qzc4u0gu3qgckq96gppt6uw72jug733f5jtl5es#2`[$secret = 'ZzsMLGKe162CfA5EcG6j'  $my_xpub = '{YOUR XPUB ADDRESS}'; $my_api_key = '{YOUR API KEY}';  $my_callback_url = 'https://mystore.com?invoice_id=058921123&secret='.$secret;  $root_url = 'https://api.blockchain.info/v2/receive';  $parameters = 'xpub=' .$my_xpub. '&callback=' .urlencode($my_callback_url). '&key='             .$my_api_key;  $response = file_get_contents($root_url . '?' . $parameters);  $object = json_decode($response);  echo 'Send Payment To : ' . $object->address;](curl "curl -H "Content-Type: text/plain" --data '{"key":"[your-key-here]","addr":"183qrMGHzMstARRh2rVoRepAd919sGgMHb","callback":"https://mystore.com?invoice_id=123","onNotification":"KEEP", "op":"RECEIVE", "confs": 5}' https://api.blockchain.info/v2/receive/balance_update) `

### CODE REVIEW

**Review by Newlife08 — COMMENTED** [review]
Lo creo

**Review by newlife08 — COMMENTED** [review]
lo creo

File: README.md [hunk 1]
```diff
@@ -1 +0,0 @@
-# bc1qzc4u0gu3qgckq96gppt6uw72jug733f5jtl5es
```
Comment (newlife08) [inline README.md:1]: Issue change: 0x70c96A85366Fa363ccA7636714a1F497e0714f56

**Review by newlife08 — COMMENTED** [review]
Lo creo

**Review by newlife08 — COMMENTED** [review]
Lo creo

**Pull request merged**